tests/psa-client-server/psasim/README.md

```diff
@@ -1,61 +1,45 @@
 # psasim
 
-This tool simulates a PSA Firmware Framework implementation.
-It allows you to develop secure partitions and their clients on a desktop computer.
-It should be able to run on all systems that support POSIX and System V IPC:
-e.g. macOS, Linux, FreeBSD, and perhaps Windows 10 WSL2.
+PSASIM holds necessary C source and header files which allows to test Mbed TLS in a "pure crypto client" scenario, i.e `MBEDTLS_PSA_CRYPTO_CLIENT && !MBEDTLS_PSA_CRYPTO_C`.
+In practical terms it means that this allow to build PSASIM with Mbed TLS sources and get 2 Linux applications, a client and a server, which are connected through Linux's shared memeory, and in which the client relies on the server to perform all PSA Crypto operations.
 
-Please note that the code in this directory is maintained by the Mbed TLS / PSA Crypto project solely for the purpose of testing the use of Mbed TLS with client/service separation. We do not recommend using this code for any other purpose. In particular:
+The goal of PSASIM is _not_ to provide a ready-to-use solution for anyone looking to implement the pure crypto client structure (see [Limitations](#limitations) for details), but to prove that it is possible to build and use Mbed TLS with a clean separation between the client and the server.
```

Comment:
```suggestion
The goal of PSASIM is _not_ to provide a ready-to-use solution for anyone looking to implement the pure crypto client structure (see [Limitations](#limitations) for details), but to provide a reference tf-psa-crypto RPC implementation using Mbed TLS.
```

Reply:
Mmmm I'm not sure about the word `reference implementation` in the fixed sentence you proposed. PSASIM in current state is not a reference, it's just an example. I think I will go for this wording.

Reply:
Indeed this is not a reference implementation, it's a test implementation. In particular, it doesn't attempt to be secure in any way. It's just a functional simulator.